crates/findex/python/cloudproof_findex/__init__.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
+from typing import Dict, List, Sequence, Union
+
 from .cloudproof_findex import *
 
+IndexedValuesAndKeywords = Dict[Union[Location, Keyword], Sequence[Union[str, Keyword]]]
```

Comment:
For my understanding, why not using a `List` here like for `SearchResults`?

Reply:
it's a trick regarding python type checking for `List` of union types and type inferences: https://mypy.readthedocs.io/en/latest/common_issues.html#variance